### PR TITLE
[Chore #159082364] Write unit tests for existing functionality

### PR DESCRIPTION
**Pull request by lytes20**
### What does this PR do?
- Adds tests to the existing functionality

### Description of Task to be completed?
- The application codebase currently has no tests. Please bring it up to sufficient coverage by writing tests for existing functionality.

### How should this be manually tested?
- `python3 manage.py test`

### Any background context you want to provide?
- I Installed Django-nose, coverage, and Django-coverage in the virtual environment
### What are the relevant pivotal tracker stories?
- [#159082364](https://www.pivotaltracker.com/story/show/159082364)
### Screenshots
screen shot 2018-07-26 at 12 37 37

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -4,4 +4,9 @@ omit =
...
     */tests/*
     */migrations/*
-    
\ No newline at end of file
+    manage.py
+
+[paths] 
+source = 
+    authors/apps/authentication/
+
```
Comment (geofrocker) [inline .coveragerc]: EOF missing here

File: authors/apps/authentication/tests/test_models.py [hunk 2]
```diff
@@ -70,28 +61,34 @@ def test_create_super_user(self):
...
.... Nam eget dui. Etiam rhoncus. Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum. Nam quam nunc, blandit vel, luctus pulvinar, hendrerit id, lorem. Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc, quis gravida magna mi a libero. Fusce vulputate eleifend sapien. Vestibulum purus quam, scelerisque ut, mollis sed, nonummy id, metus. Nullam accumsan lorem in dui. Cras ultricies mi eu turpis hendrerit fringilla. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; In ac dui quis mi consectetuer lacinia. Nam pretium turpis et arcu. Duis arcu tortor, suscipit eget, imperdiet nec, imperdiet iaculis, ipsum. Sed aliquam ultrices mauris. Integer ante arcu, accumsan a, consectetuer eget, posuere ut, mauris. Praesent adipiscing. Phasellus ullamcorper ipsum rutrum nunc. Nunc nonummy metus. Vestibulum volutpat pretium libero. Cras id dui. Aenean ut"
```
Comment (geofrocker) [inline authors/apps/authentication/tests/test_models.py]: Use a string generator method to return this string

Comment (lytes20) [reply authors/apps/authentication/tests/test_models.py]: String generator? should I use a string generator library such as this [one](https://pypi.org/project/StringGenerator/) or should i generate one using logic such as multiplying a string by a certain number or using `lambda`?

**Comment by geofrocker** [comment]
Remove the test model file from your online branch

**Pull request merged**